src/app/messages/page.tsx

```diff
@@ -31,8 +32,8 @@ export default async function MessagesPage() {
           // Note: For now I'm only handling one on one chats.
           const contactInChat = contacts?.find(contact => contact.id === userId)
 
-          // Note: (BUG) Both users in chat had the same id
           if (!contactInChat) {
+            // (BUG) If this happens is that both users in chat had the same id
```

Comment:
question: How can that happen? It could be explained in the code comment too

Reply:
it shouldn't happen! I removed the comment